From: Chris Lamb <lamby@debian.org>
Date: Thu, 21 Jun 2018 14:13:24 -0400
Subject: Make generated XPI reproducible

Without this change, modules/all-modules.txt stores details about the
sort-order of files in the filesystem used during .XPI generation.

See https://bugs.debian.org/901611
---
 util/genxpi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/util/genxpi b/util/genxpi
index d104d3a..ee17d12 100755
--- a/util/genxpi
+++ b/util/genxpi
@@ -51,7 +51,7 @@ fi
 
 cd "$targetDir"
 
-find modules -name "*.js*" > modules/all-modules.txt
+find modules -name "*.js*" | LC_ALL=C sort > modules/all-modules.txt
 
 echo "Creating ${xpiFile} file"
 
